src/lib/devices/DeviceCard/DeviceRoutes.svelte

```diff
@@ -39,7 +49,7 @@
 			<li>
 				{route}
 				{#if routesList.enabledRoutes.includes(route)}
-					<div class="btn btn-xs capitalize bg-success text-success-content mx-1">active</div>
+					<button on:click={() => {disableDeviceRouteAction(route)}} type="button" class="btn btn-xs capitalize bg-success text-success-content mx-1" data-tip="press to disable route">active</button>
```

Comment:
This button needs a `tooltip` class added to use the `data-tip` functionality.

Reply:
Sorry, this is my mistake. I fix it in new commit.